packages/web-media-box/src/dash-parser/tags/base.ts

```diff
@@ -46,24 +95,8 @@ export class Mpd extends TagProcessor {
     sharedState: SharedState,
     pendingProcessors: PendingProcessors
   ): void {
-    const id = tagInfo.tagAttributes[Mpd.ID];
-    const type = (tagInfo.tagAttributes[Mpd.TYPE] || 'static') as ManifestType;
-    const availabilityStartTime = tagInfo.tagAttributes[Mpd.AVAILABILITY_START_TIME];
-    const availabilityEndTime = tagInfo.tagAttributes[Mpd.AVAILABILITY_END_TIME];
-
-    parsedManifest.type = type as ManifestType;
-
-    if (id) {
-      parsedManifest.id = Number(id);
-    }
-
-    if (availabilityStartTime) {
-      parsedManifest.availabilityStartTime = Date.parse(availabilityStartTime);
-    }
-
-    if (availabilityEndTime) {
-      parsedManifest.availabilityEndTime = Date.parse(availabilityEndTime);
-    }
+    const attributes = formatAttributes(tagInfo.tagAttributes, MPDAttributes);
+    sharedState.attributes = attributes;
```

Comment:
I think we will eventually want these attributes to live on the `Representation` object in the parsed manifest. Therefore, I am just writing it to the state. I'll work on improving this in the future.